PhysicsTools/NanoAOD/plugins/IsoValueMapProducer.cc

```diff
@@ -255,8 +288,21 @@ void IsoValueMapProducer<pat::Photon>::doPFIsoPho(edm::Event& iEvent) const {
   fillerAll.insert(src, PFIsoAll.begin(), PFIsoAll.end());
   fillerAll.fill();
 
+  std::unique_ptr<edm::ValueMap<float>> PFIsoChgQuadraticV(new edm::ValueMap<float>());
```

Comment:
Please use `make_unique` as in https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/plugins/IsoValueMapProducer.cc#L249

Reply:
Ok, will do.